Source: "rob-fortune" @ http://ezix.org/project/ticket/521
Upstream: Reported at ticket #521
Reason: Compile failure with gcc 4.5.

diff --git a/src/core/scsi.cc b/src/core/scsi.cc
index ef49677..e72376b 100644
Index: src/core/scsi.cc
===================================================================
--- a/src/core/scsi.cc
+++ b/src/core/scsi.cc
@@ -842,7 +842,7 @@
 
           if (!controller)
           {
-            controller = node.addChild(hwNode::hwNode("scsi", hw::storage));
+            controller = node.addChild(hwNode("scsi", hw::storage));
             if (controller)
             {
               controller->setLogicalName(host_logicalname(number));
